src/pages/docs/worker-bridge.mdx

```diff
@@ -0,0 +1,452 @@
+
+### Response Mapping
+
+In the above example, we simply used `${worker.response}` to include the response as is. However, you can perform more complex transformations on the response before sending it back to the client.
+We recommend taking a quick look at the expression language before reading further.
+
+Since we are using Http protocol, worker-bridge allows you to write a record expression that consist of `status`, `headers` and `body` fields. The `status` field is the HTTP status code, `headers` is a map of HTTP headers, and `body` is the response body.
+
+Usually our `worker.response` is an Array of WASM values. This is regardless of the domain/use-case. It implies, in shopping cart example, the response is an `[[...]]`. Each object in the inner array represents a product in the cart.
+That is `worker.response[0]` will fetch the first element in the array, and this element is a sequence of product information.
+In this case, our expression of response can be something like this. Note that we are also injecting some headers in the response using expression language.
+
+```bash filename="Terminal" copy
+...
+{ status: 200, body : worker.response[0], headers: {etag: '1234567890abcdef'} }
+```
+
+Now the above expression can be embedded into the response field of API definition. Now we know that it is going to be read as an expression only if we wrap it in `${` code block
+
+```bash filename="Terminal" copy
+
+"response" : "${ { status: 200, body : worker.response[0], headers: {etag: '1234567890abcdef'} } }"
+```
+
+With Golem Cloud UI, you can easily write these expression in dedicated UI blocks making it even easier.
+
+
+Let's write a more complex example, where we need to respond back with certain details of only the first product. In this case, the response mapping will look something like this
+
+```bash filename="Terminal" copy
+{
+  status: 200,
+  body : {
+    name: worker.response[0][0].name,
+    price: worker.response[0][0].price,
+    quantity: worker.response[0][0].quantity
+  }
+  headers: { etag: '1234567890abcdef' }
+}
+```
+
+### Non-Empty Function Parameters
+In the above example, we did not pass any parameters to the function. Let's say we need to pass the user-id to the function along with a few other parameters.
+In this case, the functionParams field will contain an array of expressions that evaluate to the parameters to be passed to the function.
+
+
+```json
+{
+  "id": "shopping-cart-v1",
+  "version": "0.0.3",
+  "routes": [
+    {
+      "method": "Get",
+      "path": "/{user-id}/get-cart-contents",
+      "binding": {
+        "type": "wit-worker",
+        "template": "2696abdc-df3a-4771-8215-d6af7aa4c408",
+        "workerId": "worker-${request.path.user-id}",
+        "functionName": "golem:it/api/get-cart-contents",
+        "functionParams": [ "${request.path.user-id}", "${1}", "hardcoded" ],
```

Comment:
Once _Rib_ supports function invocations (and namespaced functions), then we can delete `functionName` and `functionParams` and just have something like `invocation: golem:it/api/get-cart-contents(request.path.user-id, 1, "hardcoded")`. i.e. the invocation is an expression. Much simpler / cleaner!

Reply:
Nice